### ISSUE TITLE
Mobile App v1.72.1 crashes when opened

### ISSUE THREAD

**Issue by traseable-solutions** [hzi-braunschweig/SORMAS-Project#9558]
### Bug Description
The mobile app (v1.72.1) crashes when opened. Sometimes it displays a message to clear the cache because it has a bug but still crashes afterwards. Sometimes it displays the message that the app has a bug.

### Steps to Reproduce
1. Download latest release ZIP file for 1.72.1
2. Install APK (from either release or debug folders in ZIP file) on mobile phone
3. Open app

### System Details

* Device: Samsung Galaxy A52
* SORMAS version: 1.72.1
* Android version: 12
* Server URL:
* User Role:

**Comment by MateStrysewske** [comment]
@traseable-solutions Does it crash immediately when opened, even if you're not logged in?

**Comment by traseable-solutions** [comment]
@MateStrysewske Yes, it crashes immediately. Splash screen opens and then it crashes immediately, sometimes with the messages I mentioned - that there is a bug or to clear the cache because of a bug.

**Comment by MateStrysewske** [comment]
@traseable-solutions Thanks, found the issue, will be fixed in 1.72.2 :)
